scala/akkahttp-low/src/main/scala/Main.scala

```diff
@@ -0,0 +1,31 @@
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model._
+import akka.stream.ActorMaterializer
+
+object Main {
+
+  def main(args: Array[String]) {
+
+    implicit val system = ActorSystem()
+    implicit val materializer = ActorMaterializer()
+
+    val requestHandler: HttpRequest => HttpResponse = {
+      case HttpRequest(GET, Uri.Path("/"), _, _, _) =>
+        HttpResponse(entity = "")
+      case HttpRequest(POST, Uri.Path("/user"), _, _, _) =>
+        HttpResponse(entity = "")
+      case HttpRequest(GET, path, _, _, _) if path.path.toString().startsWith("/user/")  =>
+        HttpResponse(entity = path.toString().split("/").last)
```

Comment:
This is not a parsing path parameter feature. If there is not a way to parse them, I cannot add akkahttp-low as a target.